### ISSUE TITLE
Update and Maintain open API docs

### ISSUE THREAD

**Issue by RakshindaAslam** [CDCgov/prime-reportstream#5284]
## Problem statement
To publish open API documents that will allow end users to discover, understand and integrate with the capabilities of the API Sender service.

## What you need to know

 [Links to documents, workflows, or list of key people and contact info]

- [Document 1](https://...)
- [Document 2](https://...)
- ...

## Acceptance criteria

- [ ] ...

## To do

- [ ] ...

**Comment by kevinhaube** [comment]
@stephenkao @jimduff-usds I think this is a great cross-team effort for Platform and Experience as we're aiming to modernize our docs to be much like true developer docs, and automation of those changes would be awesome.

**Comment by chris-kuryak** [comment]
@kevinhaube @stephenkao This looks to be more potential features regarding the "Migration of API Programmers Guide from PDF to Website," correct? And as described, this doesn't seem very actionable by the Platform team, right?

Here is what I am thinking: once the Experience team finishes its Research regarding the existing API documentation to website migration (epic #5813), we will scope the Design and Build of that migration (epic #5814). So I will move this epic under the Design and Build epic (#5814) so that it get's reviewed at that time to determine if it is in scope for that project. Then it will get refined at that time.

Any concerns about that? Should I remove the Platform label at this time so it doesn't cause confusion for that new team?

**Comment by kevinhaube** [comment]
@chris-kuryak Platform's role in this is maintenance (and honestly, ensuring the current validity) of our internal documentation files (often called our "open api spec" or "the yml files"). Experience does not own those. But as far as work to make this happen on the front-end, all that is engineered on the front-end so Platform can be removed from this ticket :)

**Comment by chris-kuryak** [comment]
@kevinhaube Gotcha, thanks for the clarification. 

So it sounds like this potential new feature would be a collaboration of Platform (API documentation maintenance and validity) and Experience (showing it on the website) teams when the time comes and the feature gets refined.

Let me know if you think I am changing/moving these issues incorrectly.

**Comment by kevinhaube** [comment]
@chris-kuryak Yeah, I'd say this has one platform dependency, which would be "verify current API docs are valid and ensure Platform has a plan for upkeep". I'd reach out and check with Avniel/Jim D on Platform to see if that's already handled; this is coming from little knowledge of what they have thus far.
